lib/vsc/utils/generaloption.py

```diff
@@ -470,6 +496,15 @@ def __init__(self, *args, **kwargs):
 
         self.environment_arguments = None
         self.commandline_arguments = None
+        self.orig_rargs = None  # copy of the original arguments
+
+    def parse_args(self, args=None, values=None):
+        """Keep a copy of the original arguments for callback / option processing"""
+        # seems like self._get_args returns a copy, but better safe then sorry here
+        self.orig_rargs = self._get_args(args)[:]
```

Comment:
I think if you make this `super(..., self)._get_args(args)`, then you can also remove the premature `return` I added in  `self._get_args`